core/ui/ControlPanel/Modals/AddPlugins.tid

```diff
@@ -5,7 +5,7 @@ subtitle: {{$:/core/images/download-button}} {{$:/language/ControlPanel/Plugins/
 
 \define install-plugin-button()
 <$button>
-<$action-sendmessage $message="tm-load-plugin-from-library" url={{!!url}} title={{$(assetInfo)$!!original-title}}/>
```

Comment:
@Jermolene ...sorry, no idea what I was thinking ...now I see it being used "bright as daylight" ...reverted that change